spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -858,18 +969,21 @@ def get_buffered_sdram(self, variable, vertex_slice, n_machine_time_steps):
         :param ~pacman.model.graphs.common.Slice vertex_slice:
         :param int n_machine_time_steps:
             how many machine time steps to run for
+        :param int max_rewires_per_ts: the max possible rewires per timestep
         :return: data size
         :rtype: int
         """
         # Per timestep variables can't be done at a specific rate
         if variable in self.__per_timestep_variables:
-            item = self.get_buffered_sdram_per_record(variable, vertex_slice)
+            item = self.get_buffered_sdram_per_record(
+                variable, vertex_slice, max_rewires_per_ts)
```

Comment:
If max_rewires_per_ts is not part of the "variables" here, it won't end up being called here, and can instead be treated separately below this loop.  This will open up for future usage should another variable like this end up being needed.